dotCMS/src/integration-test/java/com/dotmarketing/tag/business/TagAPITest.java

```diff
@@ -640,54 +643,68 @@ public void updateTagReferences() throws Exception{
 				//Do nothing...
 			}
 
-			TagFactory tagFactory = FactoryLocator.getTagFactory();
```

Comment:
`isInodeIndex` and `thread.sleep` should be able to be removed

Reply:
done